app/main/views/signup.py

```diff
@@ -81,7 +80,7 @@ def send_seller_signup_email():
 
 
 @main.route('/signup/create-user/<string:token>', methods=['GET'])
-def render_create_application(token, data={}, errors=None):
+def render_create_application(token={}, data={}, errors=None):
```

Comment:
These defaults need changing, v bad things happen in python when you set parameter defaults to mutable objects.

Instead set defaults to None and then do:

token = token or {}

or similar at the start of the method